specification/search/data-plane/Azure.Search/preview/2024-11-01-preview/searchindex.json

```diff
@@ -0,0 +1,3220 @@
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "debug",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "disabled",
+              "semantic",
+              "vector",
+              "queryRewrites"
+            ],
+            "x-ms-enum": {
+              "name": "QueryDebugMode",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "disabled",
+                  "name": "Disabled",
+                  "description": "No query debugging information will be returned."
+                },
+                {
+                  "value": "semantic",
+                  "name": "Semantic",
+                  "description": "Allows the user to further explore their reranked results."
+                },
+                {
+                  "value": "vector",
+                  "name": "Vector",
+                  "description": "Allows the user to further explore their hybrid and vector query results."
+                },
+                {
+                  "value": "queryRewrites",
+                  "name": "QueryRewrites",
+                  "description": "Allows the user to explore the list of query rewrites generated for their search request."
+                }
+              ]
+            },
+            "description": "Enables a debugging tool that can be used to further explore your search results.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "queryLanguage",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "en-us"
+            ],
+            "x-ms-enum": {
+              "name": "QueryLanguage",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Query language not specified."
+                },
+                {
+                  "value": "en-us",
+                  "name": "EnUs",
+                  "description": "English (US)"
+                }
+              ]
+            },
+            "description": "The language of the query.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "speller",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "lexicon"
+            ],
+            "x-ms-enum": {
+              "name": "Speller",
```

Comment:
The name for the other `Speller` definition is `QuerySpellerType`, could we make that change here as well.